### PR TITLE
Refactor of classes

### PR DESCRIPTION
**Pull request by maciekbanas**

### CODE REVIEW

File: R/plot_repos.R [hunk 1]
```diff
@@ -11,15 +11,11 @@
...
-    stop("You have to first construct your repos data.frame with 'get_repos' function.",
-      call. = FALSE
-    )
-  }
 
-  repos_dt <- gitstats_obj$repos_dt
+  repos_dt <- gitstats_obj$show_repos()
 
-  repos_to_plot <- head(repos_dt, repos_n)
+  repos_to_plot <- repos_dt[order(last_activity_at)]
```
Comment (marcinkowskak) [inline R/plot_repos.R:17]: While going through example_workflow.R file the code crashes in this line and I'm not able to plot repos. It looks like it has some issues with the way you declare sorting of the table.

image

**Review by marcinkowskak — CHANGES_REQUESTED** [review]

File: R/plot_repos.R (hunk 1 above)
Comment (maciekbanas) [reply R/plot_repos.R:19]: I get this error when the input to plot function is empty (so when I try to plot repos before pulling them to `GitStats`) - can you check you've pulled first repos? 
I pushed NULL handler to plots right now - so when the repos/commits are empty, the error pops out with the appropriate message.

Comment (marcinkowskak) [reply R/plot_repos.R:19]: After some changes I was able to print plot. Looks good but if I could suggest something: can we remove the name of the organization on the Y axis? It's doubled information with the color: 
image

It's not critical though.

**Review by marcinkowskak — APPROVED** [review]
Approving PR as the issue with labels on plot is not critical and can be taken care of later.

**Pull request merged**